frame/election-provider-multi-phase/src/mock.rs

```diff
@@ -254,6 +254,10 @@ impl pallet_balances::Config for Runtime {
 	type MaxReserves = ();
 	type ReserveIdentifier = [u8; 8];
 	type WeightInfo = ();
+	type FreezeIdentifier = ();
```

Comment:
Have you seen https://github.com/paritytech/substrate/pull/13454, and any opinions on it? It would have made the diff for this PR significantly smaller, assuming that you would think it is safe to use these defaults everywhere.

Reply:
How far off merge is it?

Reply:
I'd guess a week or two, but @sam0x17 can verify. 

The base PR can probably be merged soon, without using it extensively.